web/src/beta/lib/reearth-ui/components/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, ReactNode } from "react";
+
+import { Icon, IconName, Typography } from "@reearth/beta/lib/reearth-ui";
+import { styled, useTheme } from "@reearth/services/theme";
+
+import { Items } from "../PopupMenu";
+
+export type ItemsProp = {
```

Comment:
```suggestion
export type BreadcrumbItem = {
```
I think it's better to have a clear naming since it's been exported. No `prop` need here since it's not a prop of a function.